lib/charms/opensearch/v0/opensearch_plugin_manager.py

```diff
@@ -58,14 +56,6 @@
         "class": OpenSearchKnn,
         "config": "plugin_opensearch_knn",
     },
-    "repository-s3": {
```

Comment:
from now on, `OpenSearchBackup` classes will handle these plugins, as the place we read the config options in a given relation databag changes according to the app being main (hence reading from s3 relation) or non-main (hence, reading from secrets in the peer-cluster relation).